src/EthMultiVault.sol

```diff
@@ -261,7 +270,12 @@ contract EthMultiVault is IEthMultiVault, Initializable, ReentrancyGuardUpgradea
 
     /// @dev set admin
     /// @param admin address of the new admin
-    function setAdmin(address admin) external onlyAdmin {
+    /// @notice Requires new admin to 'confirm' the timelocked operation
+    //  @dev Old admin may still cancel this before timelock duration if desired
+    function setAdmin(address admin) external {
+        if (msg.sender != admin) {
```

Comment:
3.9 - EthMultiVault Could Implement a Bilateral Transfer of Administrator Rights

We have added this requirement to ensure that the caller or recipient has been set as the admin.